File: SumOfFactor/FactorSum.js
// Find factorial number and factors and sum of factor

// n = parseInt(prompt("Enter number: "))
n = 5
sum = 0
fact = 1
temp = []
for (let i = 1; i <= n; i++) {
    if (n % i == 0) {
        sum += i
        temp.push(i);
    }
    fact *= i
}
console.log(`Factorial of ${n} = ${fact}\nFactors of ${n} = ${temp} and sum = ${sum}`)

File: SumOfFactor/FactorSum.py
# WAP to find sum of factors and factorial too.

n = int(input("Enter number: "))
sum = 0
fact = 1
j = []
for i in range(1, n + 1):
    if n % i == 0:
        sum += i
        j.append(i)
    fact *= i
print(f"Factorial of {n} = {fact}\nFactor of {n} = {j} & sum of Factors = {sum}")
